libs/langgraph/langgraph/pregel/types.py

```diff
@@ -92,6 +92,8 @@ class StateSnapshot(NamedTuple):
     """Config used to fetch the parent snapshot, if any"""
     tasks: tuple[PregelTask, ...]
     """Tasks to execute in this step. If already attempted, may contain an error."""
+    subgraph_state_snapshots: Optional[dict[str, "StateSnapshot"]] = None
+    """State snapshots of subgraphs represented as a mapping from thread ID suffix to snapshot."""
```

Comment:
I think you mean checkpoint namespace not thread suffix?

Reply:
oh great catch that was a leftover from a few weeks back, pre namespacing